packages/dashboard-backend/src/devworkspace-client/services/api/workspace-api.ts

```diff
@@ -188,10 +187,19 @@ export class DevWorkspaceApi implements IDevWorkspaceApi {
       } else {
         callbacks.onError(`Error: Unknown type '${type}'.`);
       }
-    }, error => {
-      console.error('Watch in namespace error:', error);
-      const message = typeof (error) === 'string' ? error : 'Unknown type error.'
-      callbacks.onError(message);
+    }, (error: any) => {
+      let message;
```

Comment:
Could we use our ```getMessage(error: unknown): string``` here?
https://github.com/eclipse-che/che-dashboard/blob/main-7.37/packages/common/src/helpers/errors.ts#L21

Reply:
good idea

Reply:
The only problem I noticied with it:
https://github.com/eclipse-che/che-dashboard/blob/main-7.37/packages/common/src/helpers/errors.ts#L62
```
 return 'Unexpected error. Check DevTools console and network tabs for more information.'
```
which is good error for frontend but not for backend, which is my case.

leaving as is and will think how we can handle it